app/models/teacher.rb

```diff
@@ -17,7 +17,10 @@ class Teacher < ApplicationRecord
   has_many :induction_periods
   has_one :first_induction_period, -> { order(started_on: :asc) }, class_name: "InductionPeriod"
   has_one :last_induction_period, -> { order(started_on: :desc) }, class_name: "InductionPeriod"
+  has_one :ongoing_induction_period, -> { ongoing }, class_name: "InductionPeriod"
```

Comment:
There are a few places we might want to use this. Do we want to do that here or later? 💭

Reply:
I'm not in tomorrow but FWIW, depends how many there are - might be nicer to get this in then create follow ups?

Reply:
Opened #919